package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@sentry-internal/global-search",
   "description": "JavaScript library and helper utilities for searching Sentry sites via Algolia.",
-  "version": "0.1.3",
+  "version": "0.2.0",
```

Comment:
I figured we should at least increment the minor version with upgrading Webpack, maybe this should be 1.0.0 though